Backbase/variants
# Add unit tests to SpecHelper

### What does this PR do
 - Unit test `SpecHelper`
 - Adjustments for code quality manual validation

### How can it be tested
 - Run `make validation` should return no error and output the unit test coverage.

### Task
part of #82 

### Overall Unit Test Coverage
Tested 732/1647 statements
Test Coverage: 44.44%

### Checklist:

- [x] I ran `make validation` with success
